src/Spryker/Client/RabbitMq/Model/Helper/QueueEstablishmentHelperInterface.php

```diff
@@ -16,15 +16,15 @@ interface QueueEstablishmentHelperInterface
      * @param \PhpAmqpLib\Channel\AMQPChannel $channel
      * @param \Generated\Shared\Transfer\RabbitMqOptionTransfer $queueOptionTransfer
      *
-     * @return array
+     * @return mixed[]
      */
-    public function createQueue(AMQPChannel $channel, RabbitMqOptionTransfer $queueOptionTransfer);
+    public function createQueue(AMQPChannel $channel, RabbitMqOptionTransfer $queueOptionTransfer): array;
 
     /**
      * @param \PhpAmqpLib\Channel\AMQPChannel $channel
      * @param \Generated\Shared\Transfer\RabbitMqOptionTransfer $queueOptionTransfer
      *
-     * @return \Generated\Shared\Transfer\RabbitMqOptionTransfer
+     * @return void
```

Comment:
But why? :)

Reply:
Because it is the lie.

https://github.com/spryker/rabbit-mq/blob/master/src/Spryker/Client/RabbitMq/Model/Helper/QueueEstablishmentHelper.php#L46